src/pages/Governance/Stake/Index.tsx

```diff
@@ -24,11 +23,21 @@ import {isValidAccountAddress} from "../../utils";
 export function StakePage() {
   const [stakingAmount, setStakingAmount] = useState<string>("");
   const [operatorAddr, setOperatorAddr] = useState<string>("");
+  const [operatorAddrIsValid, setOperatorAddrIsValid] = useState<boolean>(true);
   const [voterAddr, setVoterAddr] = useState<string>("");
+  const [voterAddrIsValid, setVoterAddrIsValid] = useState<boolean>(true);
 
   const {submitStake, transactionResponse, clearTransactionResponse} =
     useSubmitStake();
 
+  useEffect(() => {
+    setOperatorAddrIsValid(true);
+  }, [operatorAddr]);
+
+  useEffect(() => {
+    setVoterAddrIsValid(true);
```

Comment:
not completely sure why you set it to `true` every time the `voterAddr` is changed? same goes to the upper useEffect and the one on `VoteButtons` file

Reply:
because once user starts to change the input (typing or copy-pasting new things), we want to make the error message disappear. The idea is to only validate input when the user tries to submit a vote.